src/Search.php

```diff
@@ -216,20 +216,24 @@ function ( $args ) {
 		);
 		// Because of how the search is currently set up (using admin-ajax) this ElasticPress filter was not being
 		// applied for subsequent page loads, only for the initial one.
-		if ( ( ! isset( $_GET['orderby'] ) || '_score' === $_GET['orderby'] ) && wp_doing_ajax() ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
-			add_filter(
-				'ep_formatted_args',
-				[ new \ElasticPress\Feature\Search\Search(), 'weight_recent' ],
-				10,
-				2
+		$ep_exists = class_exists( \ElasticPress\Feature\Search\Search::class )
```

Comment:
On production we never run without ElasticSearch, so I'm not sure if we should be making the search code even more complex just so we can support running it locally without it :thinking:

Reply:
I mean, if the test is failing because it's missing ES, that means it's checking a local environment which is significantly different from what production runs. Wouldn't it make more sense to ensure the ES plugin is included in localdev and during the test?

Reply:
I had a my regular argumentation ready for this discussion, but then I just found out that configuring ElasticPress with an empty url for host settings removes the admin panel error message and generates 0 exception, which were the 2 main things I was fighting against. Thanks for the push :partying_face:  closing this, I'm gonna open another one keeping ElasticPress enabled and just modifying config :ok_hand: